### ISSUE TITLE
[BLOCKED] Campaign Landing Page request: Veteran Childcare Assistance Program

### ISSUE THREAD

**Issue by stefaniefgray** [department-of-veterans-affairs/va.gov-cms#16802]
## Related links
[2024-11-14] [Fran] Still on hold
March 2024 JIRA: https://va-gov.atlassian.net/browse/VAHELP-6902
Oct 2024 JIRA: https://va-gov.atlassian.net/browse/VAHELP-8086
Existing CLP node: https://prod.cms.va.gov/initiatives/veteran-childcare-assistance-program

## Status 
- **[2024-05-08]**  [Ian S] The Campaign Landing page is sitting in DRAFT mode [here](https://prod.cms.va.gov/initiatives/veteran-childcare-assistance-program). Ian just now [reached out again to Justin Warren](https://va-gov.atlassian.net/browse/VAHELP-6902) reminding him of the status of this.

- **[2024-03-20]** The last info Mikki N received: they were going to publish info on this program via the facility site that they are piloting it at. Once the pilot expands, they may come back around to this. So this may be on hold for awhile.
- **[2024-02-28]** Stefanie commented on 2/6: Waiting on draft homepage edits -- https://prod.cms.va.gov/initiatives/veteran-childcare-assistance-program

- **[2024-01-18]** Awaiting more information from Justin, who mentioned week of 1/18 that he is gathering information and should be able to update the ticket soon.

## Description
Use this issue to request a new Campaign Landing Page on behalf of a requesting team. 

## Intake
- [x] What triggered this runbook? (Help desk ticket, Product team, Slack thread, etc)
Trigger: Jira ticket

- [x] Link to associated JIRA help desk ticket (if applicable)
Help desk ticket: https://va-gov.atlassian.net/browse/VAHELP-6902

- [x] Vanity URL requested: Unknown

- [x] Name of submitter (if applicable)
Submitter: Justin Warren, VHA Digital Media

**1. The department publishing the campaign:**
The Office of Womens Health – Veteran Child Care Assistance Program 

**2. The official title of the campaign, for example "Veterans Experience Action Center":**
VA Kids Care Opening 

**3. Who the editor(s) will be for the Campaign Landing Page and any appropriate stakeholders for awareness:**
Brandy Morrison 

**4. The goals/outcomes you are looking to achieve with the campaign. For example: "Connect Veterans and their families to local and VA resources. Rapidly process their claims and enroll them In VA health care."** 
Connect Veteran caretakers to available child care resources while attending qualifying VA health care appointments at a participating VA facility. 

**5. Outcome success measurement: How you will be measuring progress and success. For example: Increase in flu-shot appointments (note: "Page views" is not a generally accepted success measurement)** 
- Utilization numbers at first VA Kids Care
- Increased media interest around VA Kids Care going live
- Increased site interest from VA health care systems that have not yet committed to VA Kids Care. 
- Increased participation and interaction from sites who have already committed to VA Kids Care 

**6. Target Audience(s). For example: Veterans, Families, Survivors** 
Veterans 

**7. Campaign start/end dates. For example: 7/30 - 11/22** 
Dependent on the Fresno go-live. Targeting a tentative date of April 26, 2024. However, we recognize that this date could shift in either direction. 

**8. Seasonal Campaign (e.g. Get Your Flu Vaccine, Pride Month):**
No

**9. What you plan to do when you retire the page. For example "We will redirect the URL to the VA prevention website"** 
It is our hope that by the time the Campaign Landing Page retires, VCAP will have its own permanent VA webpage that we can point stakeholders to, ensuring that we are a top influencer in Search Engine Optimization. This will be crucial as we roll out the Direct Veteran Reimbursement component of the VCAP program as Veterans will need to file a claim. 

**10. The vanity URL you’d like to use for this Campaign Landing Page? (For example: https://www.va.gov/initiatives/flu )**
[www.va.gov/VAKidsCare](http://www.va.gov/VAKidsCare)

[VAHELP-6902]: https://va-gov.atlassian.net/browse/VAHELP-6902?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**CLP NODE:** https://prod.cms.va.gov/initiatives/veteran-childcare-assistance-program

## Acceptance criteria
  
- Request comes in, via helpdesk request or direct to PO, or potentially in a Github issue.
- [x] If not in a helpdesk request, redirect to Helpdesk or create a Helpdesk issue using email support@va-gov.atlassian.net or URL https://prod.cms.va.gov/help
Helpdesk will use #sitewide-public-websites channel to request approvals
- [x] In helpdesk ticket, @ mention Dave Conlon to review/approve the creation / existence / description of the CLP. Example text: 
> Hi Dave, Helpdesk has received a request for a new Campaign Landing Page. Can you confirm that this CLP is approved?

#### Helpdesk steps
- [x] Helpdesk or any Drupal Admin can create the CLP node. Clone the [CLP template node](https://prod.cms.va.gov/node/16512) 
<insert_CLP_node_link>
- [x] If a new CMS editor user is needed, Helpdesk or any Drupal admin can create them and assign proper permissions according to [CLP CMS account administration policy](https://prod.cms.va.gov/help/cms-account-admin-policies/clp-cms-account-administration-policy).
- [x] Send the editor a link to the KB article: [self-guided Campaign Landing Page training](https://prod.cms.va.gov/help/campaign-landing-pages/how-to-manage-campaign-landing-pages)
- [x] Create a post in DSVA slack channel #sitewide-public-websites, and @ mention Jill Adams and Fran Cross: "New CLP requested and approved, for your awareness:" with a link to this Github issue for next steps.

#### Public Websites steps
- [x] Create a [Vanity URL redirect issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&template=redirect-request.md&title=Redirect+Request), IA _must_ review & approve URL requests.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/74256

- [ ] @mention Mikki Northuis in IA (#sitewide-content-ia) with link to Github issue, to provide Vanity URL request feedback. IA _must_ review & approve URL requests.

- [ ] When approved by IA, execute the steps of the URL change request ticket from step 2 above.

- [ ] When redirect is verified in Production, close Redirect ticket as well as this ticket.

**Comment by jilladams** [comment]
Slack thread for Dave approval surfaced some additional questions he'd like Justin to clarify: 
https://dsva.slack.com/archives/C52CL1PKQ/p1704913352661799?thread_ts=1704909955.736309&cid=C52CL1PKQ. I think these belong in this issue template, so I'll add that for future and have added a section for them in the body of this ticket

**Comment by FranECross** [comment]
Pinged Stefanie about the questions Jill added to this story (and template) that will need to be answered before Dave and review/approve (in the Slack thread noted above).

**Comment by FranECross** [comment]
Stefanie checked with Justin, and this is on his radar to do soon.

**Comment by stefaniefgray** [comment]
@jilladams @FranECross Here's Justin's response:

**1. The department publishing the campaign:**
The Office of Womens Health – Veteran Child Care Assistance Program 

**2. The official title of the campaign, for example "Veterans Experience Action Center":**
VA Kids Care Opening 

**3. Who the editor(s) will be for the Campaign Landing Page and any appropriate stakeholders for awareness:**
Brandy Morrison 

**4. The goals/outcomes you are looking to achieve with the campaign. For example: "Connect Veterans and their families to local and VA resources. Rapidly process their claims and enroll them In VA health care."** 
Connect Veteran caretakers to available child care resources while attending qualifying VA health care appointments at a participating VA facility. 

**5. Outcome success measurement: How you will be measuring progress and success. For example: Increase in flu-shot appointments (note: "Page views" is not a generally accepted success measurement)** 
- Utilization numbers at first VA Kids Care
- Increased media interest around VA Kids Care going live
- Increased site interest from VA health care systems that have not yet committed to VA Kids Care. 
- Increased participation and interaction from sites who have already committed to VA Kids Care 

**6. Target Audience(s). For example: Veterans, Families, Survivors** 
Veterans 

**7. Campaign start/end dates. For example: 7/30 - 11/22** 
Dependent on the Fresno go-live. Targeting a tentative date of April 26, 2024. However, we recognize that this date could shift in either direction. 

**8. Seasonal Campaign (e.g. Get Your Flu Vaccine, Pride Month):**
No

**9. What you plan to do when you retire the page. For example "We will redirect the URL to the VA prevention website"** 
It is our hope that by the time the Campaign Landing Page retires, VCAP will have its own permanent VA webpage that we can point stakeholders to, ensuring that we are a top influencer in Search Engine Optimization. This will be crucial as we roll out the Direct Veteran Reimbursement component of the VCAP program as Veterans will need to file a claim. 

**10. The vanity URL you’d like to use for this Campaign Landing Page? (For example: https://www.va.gov/initiatives/flu )**
[www.va.gov/VAKidsCare](http://www.va.gov/VAKidsCare)

**Comment by jilladams** [comment]
CLP approved to move forward: https://dsva.slack.com/archives/C52CL1PKQ/p1706017504405899?thread_ts=1704909955.736309&cid=C52CL1PKQ

**Comment by jilladams** [comment]
@stefaniefgray @TroyA6 reassigning to you for the Helpdesk steps to create the CLP node, share with the Editor, and close your Helpdesk ticket. Once that's done, this ticket can be assigned back to me / Fran.

**Comment by stefaniefgray** [comment]
Waiting on draft homepage edits -- https://prod.cms.va.gov/initiatives/veteran-childcare-assistance-program

**Comment by FranECross** [comment]
@stefaniefgray Just checking back on this ticket to see if there's any movement on approval of the homepage edits? Thanks!

**Comment by jilladams** [comment]
@FranECross Stefanie transitioned off the project when the new CMS team took over. The new helpdesk contacts are @ian-sears and Antwuan Jones (not sure his Github handle.) 

Looking at the redirect ticket linked from here (https://github.com/department-of-veterans-affairs/va.gov-team/issues/74256) , it looks like this sits right now with @mnorthuis for IA feedback on the vanity URL request.

**Comment by mnorthuis** [comment]
For this one, the last information I got on this, is that they were going to publish info on this program via the facility site that they are piloting it at. Once the pilot expands, they may come back around to this. So this may be on hold for awhile.

**Comment by ian-sears** [comment]
Just updating this. The CMS Help Desk contacts are me @ian-sears and Troy Griffin @TroyA6 

And the Campaign Landing page is sitting in DRAFT mode here
https://prod.cms.va.gov/initiatives/veteran-childcare-assistance-program

I've just now reached out again to Justin Warren reminding him of the status of this.
https://va-gov.atlassian.net/browse/VAHELP-6902

**Comment by ian-sears** [comment]
Followed up with Justin Warren today asking for status/decision/progress on this from their end. Awaiting reply from Justin Warren at the CMS Help Desk in Jira [VAHELP-8086](https://va-gov.atlassian.net/browse/VAHELP-8086)

@department-of-veterans-affairs/platform-cms-helpdesk fysa

**Comment by ian-sears** [comment]
UPDATE from Justin Warren just now:
"Thanks for following up. Setting up this page has been put on hold for now."
We are asking hi "postponed for how long?" Awaiting reply.

**Comment by jilladams** [comment]
Helpdesk ticket was closed by the requester with a note: 
> This work is paused, so we are closing this ticket. Justin Warren will give us direction when this work begins again. 

Closing this as well.
